<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>185</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>185</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>185</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Password Generator</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="gen_sym_num_label">
        <property name="text">
         <string>Длина пароля:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="password_length">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum">
         <number>4</number>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="include_special">
      <property name="text">
       <string>Включить специальные символы (!@#%_$~)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="include_digits">
      <property name="text">
       <string>Включить цифры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="include_lowercase">
      <property name="text">
       <string>Включить строчные буквы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="include_uppercase">
      <property name="text">
       <string>Включить заглавные буквы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="dialog_buttons">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
